release-notes/6.0/supported-os.json

```diff
@@ -338,7 +338,6 @@
                         "11-21h2-e",
                         "10-21h2-e-lts",
                         "10-21h2-iot-lts",
-                        "10-20h2-e-lts",
```

Comment:
Windows 10 20H2 was only a semi-anual version and never a LTS. Therefore there was also no information on EndOfLife.date.

Reply:
You are right. Not sure what happened there. Likely a copy/paste error.

Reply:
Shouldn't `10-20h2-e` and `10-20h2-2` be added to the unsupported list?

Reply:
10-20h2-e and 10-20h2-w are already on the unsupported list. Not sure what you mean by 10-20h2-2.